OxfordBodleian/Juel-Jensen/BDLaethe33.xml

```diff
@@ -226,6 +233,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="right">18</dim>
                                         <dim type="left">14</dim>
                                     </dimensions>
+                                    <note> The characters per line are 9-23.</note>
+                                    <ab type="pricking">Ruling and pricking are visible. Both primary and ruling pricks are visible.</ab>
```

Comment:
I am not familiar with the use of "primary pricks" and it seems like it has only been used twice by Denis?

Reply:
I have inherited this terminology from the Ethio-SPaRe cataloguing practice (as @SusanneHummel has done). In his "Pricking and Ruling in Ethiopic Manuscripts: an Aid for Dating?", Denis (COMSt Bulletin 1/2, 2015, p. 94) speaks of "vertical pricks" (serving for impressing the vertical bounding lines which delimit the text column) and "text pricks" (serving for impressing the horizontal text lines). If we are unhappy with the adopted terms, we might stick to that terminology, which furthermore finds confirmation in a printed contribution.

Reply:
I created a list of points and adding them to a file entitled "meeting agenda" in our shared folder, please feel free to add things you want to discuss in our next meeting there.

Reply:
I checked in Ligatus; there is no "pricking" at all ...